src/components/basket-item/basket-item.jsx

```diff
@@ -4,27 +4,69 @@ var React = require('react');
 module.exports = React.createClass({
   propTypes: {
     currencySymbol: React.PropTypes.string,
+    freeText: React.PropTypes.string,
+    removeText: React.PropTypes.string,
     price: React.PropTypes.number,
+    title: React.PropTypes.node,
+    toggleDescription: React.PropTypes.bool,
+    handleRemove: React.PropTypes.func,
     children: React.PropTypes.node
   },
+  getInitialState: function() {
+    return {
+      descriptionVisibility: !this.props.toggleDescription
+    };
+  },
   getDefaultProps: function() {
     return {
-      currencySymbol: '£'
+      title: null,
+      price: null,
+      handleRemove: null,
+      currencySymbol: '£',
```

Comment:
I would agree... but it looks like internationalisation has not been covered on UI-Toolkit at all yet. Should this be something I introduce now? Feels like it's beyond the scope of my task.